modules/editor/EditorManager.ts

```diff
@@ -22,9 +27,18 @@ export const EditorManager = types
 
     clear() {
       self.messages.clear()
+      this.add()
+    },
+
+    add() {
       self.messages.push(MessageModel.create())
     },
 
+    delete(message: MessageLike) {
+      message.delete()
+      destroy(message)
+    },
+
```

Comment:
Don't implement deleting now, and adding/removing embeds from the editor should be done with the form system.
```suggestion
    clear() {
      self.messages.clear()
      self.messages.push(MessageModel.create())
    },
```